.appveyor.yml

```diff
@@ -13,10 +13,13 @@ branches:
 
 environment:
   matrix:
-    - COMPILER: MINGW
-      TASK: r-package
-    - COMPILER: MSVC
-      TASK: r-package
+    - TASK: r-package
+      R_VERSION: 4.0
+      COMPILER: MINGW
+      TOOLCHAIN: MSYS
+    - TASK: r-package
+      R_VERSION: 4.0
+      COMPILER: MSVC
       APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2019
     - COMPILER: MSVC
```

Comment:
Please use the same order for variables inside jobs: either `TASK`-first or `COMPILER`-first for better readbility.

Reply:
ok

Reply:
fixed in https://github.com/microsoft/LightGBM/pull/3065/commits/c65a91078d23001962a956abfc915ad7ebfba155